cert_deployer/deploy.py

```diff
@@ -77,7 +77,7 @@ def _compile_contract(self):
             opt = json.loads(raw_opt)
         #opt["sources"]["ResearchCertificate.sol"]["content"] = source_raw
         #compiled_sol = compile_source(source_raw)
-        compiled_sol = compile_files([tools.get_contr_path()], output_values=["abi", "bin"], solc_version="v" + self.app_config.compiler_version)
+        compiled_sol = compile_files([tools.get_contr_path()], output_values=["abi", "bin"])
```

Comment:
This is the main change. Removed `solc_version` since it was causing the error. I don't know if you can reproduce this. Removing solved the issue.